sustainml_modules/sustainml_modules/sustainml-wp4/data.html

```diff
@@ -0,0 +1,265 @@
+                    <button class="response-button" id="upload-button">Upload</button>
+                    <div id="file-details" class="hidden">
+                        <h3>File Details</h3>
+                        <div class="table-container">
+                            <table id="file-rows-table" class="centered-table"></table>
+                        </div>
+                        <p id="empty-values"></p>
+                        <div id="column-selection" class="hidden">
+                            <p style="color: red;">Confirm your selection</p>
+                            <button class="response-button" id="confirm-columns-button">Confirm</button>
+                        </div>
+                        <div id="confirmation-box" class="hidden">
+                            <div class="modal-content">
+                                <p id="selected-columns-message"></p>
+                                <button class="response-button" id="confirm-yes-button">Yes</button>
+                                <button class="response-button" id="confirm-no-button">No</button>
+                            </div>
+                        </div>
+                        <button class="response-button hidden" id="see-more-button">See More</button>
+                    </div>
+                </div>
+                <div class="vertical-line"></div>
+                <div class="split-item" id="search-section">
+                    <h2>Search for Data</h2>
+                    <div class="input-section" id="search-modality-section">
+                        <p>What is your data modality?</p>
+                        <select id="modality-select">
+                            <option value="">Select Modality</option>
+                            <option value="image_video">Image/Video</option>
+                            <option value="text">Text</option>
+                            <option value="sensor">Sensor</option>
+                            <option value="audio">Audio</option>
+                        </select>
+                        <button class="response-button" id="submit-modality">Search</button>
+                    </div>
+
+                    <div class="input-section hidden" id="search-domain-section">
+                        <p>What is your domain of application?</p>
+                        <input type="text" id="domain-input" placeholder="Enter domain of application">
+                        <button class="response-button" id="submit-domain">Search</button>
+                    </div>
+                    <div class="input-section hidden" id="search-results-section">
+                        <h3>Search Results</h3>
+                        <ul id="search-results-list"></ul>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <script>
+        document.addEventListener('DOMContentLoaded', function() {
```

Comment:
I think it would be interesting to group the code under `script` in a separate `.js` file and just include it here, what do you think ?

Reply:
Was not able to effectively migrate it to a separate script